tests/test_hmc.py

```diff
@@ -150,27 +155,30 @@ def logprob_fn(y):
             None,
             None,
         ],
-        non_sequences=0.5,
-        n_steps=2000,
+        non_sequences=1.0,
```

Comment:
Does it not pass with a smaller step size?

Reply:
It does, but it runs faster with larger step sizes (which is expected).

Reply:
It does. Here I adapted the step size manually to get an average acceptance probability of 0.8 (target in the window adaptation warmup); it also runs slightly faster.